lib/Controller/PushController.php

```diff
@@ -100,7 +100,7 @@ public function registerDevice(string $pushTokenHash, string $devicePublicKey, s
 		if (
 			!filter_var($proxyServer, FILTER_VALIDATE_URL) ||
 			\strlen($proxyServer) > 256 ||
-			!preg_match('/^(https\:\/\/|http\:\/\/localhost(\:\d{0,5})?\/)/', $proxyServer)
+			!preg_match('/^(https\:\/\/|http\:\/\/(localhost|host\.docker\.internal)(\:\d{0,5})?\/)/', $proxyServer)
```

Comment:
```suggestion
			!preg_match('/^(https\:\/\/|http\:\/\/(localhost|[a-z0-9]*\.(internal|local))(\:\d{0,5})?\/)/', $proxyServer)
```
Let's then allow any internal or local domain while on it.

Reply:
I slightly changed your suggestion because it didn't allow `.` and `-`.

Reply:
oopsie

Reply:
now it's there